cmds/rush/lua.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2012 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Time process execution.
```

Comment:
yeah this is a very rough job because I wanted some feedback. I agree the state should persist -- easily done, we just put it in the init function.

So do we do this? Lua is certainly a popular language.